src/components/Header/SearchBar/SearchBar.tsx

```diff
@@ -103,30 +105,39 @@ export const SearchBar = ({
           'mx-24 mt-24 md:m-0': displayLocation !== 'homepage',
         })}
       >
-        <input
-          type="text"
-          ref={textInput}
-          placeholder="Search"
-          className={cn(searchInput)}
-          value={query}
-          onChange={handleSearch}
-          style={{ ...extraInputStyle }}
-        />
-        <Icon name="icon-gui-search" size="24px" additionalCSS="absolute left-16 top-12" />
-        {!isInFocus && (
-          <div className="absolute right-16 top-12 hidden lg:flex items-center justify-end">
-            <KeyIcon className="mr-4">{keyIcon}</KeyIcon>
-            <KeyIcon className="pt-2">K</KeyIcon>
-          </div>
+        {externalScriptsData.inkeepEnabled ? (
+          <InkeepSearchBar
+            className={`${cn(searchInput)} ${cn(searchInputNoPadding)}`}
+            extraInputStyle={{ ...extraInputStyle }}
+          />
+        ) : (
+          <>
+            <input
+              type="text"
+              ref={textInput}
+              placeholder="Search"
+              className={cn(searchInput)}
+              value={query}
+              onChange={handleSearch}
+              style={{ ...extraInputStyle }}
+            />
+            <Icon name="icon-gui-search" size="24px" additionalCSS="absolute left-16 top-12" />
+            {!isInFocus && (
```

Comment:
We should present these conditional renderings as ternaries with `null` on the other end, as passing `null` is more beneficial to the renderer than `false`.

Reply:
It's existing logic, it's just been shifted down into the new ternary. Minimising blast radius by only changing what is necessary really